pilot/cmd/pilot-agent/options/agent.go

```diff
@@ -44,6 +44,9 @@ func NewAgentOptions(proxy *model.Proxy, cfg *meshconfig.ProxyConfig) *istioagen
 		ServiceNode:                 proxy.ServiceNode(),
 		EnvoyStatusPort:             envoyStatusPortEnv,
 		EnvoyPrometheusPort:         envoyPrometheusPortEnv,
+		TdInterceptionPort:          tdInterceptionPortEnv.Get(),
```

Comment:
I don't see where this is actually used..?

Reply:
I don't think we need to define it in code then. Anything with ISTIO_META_ is passed

Reply:
Removed this,